turbinia/workers/__init__.py

```diff
@@ -123,10 +121,8 @@ def __str__(self):
 
   def setup(self, task):
     """Handles initializing task based attributes, after object creation.
```

Comment:
Strange, I guess YAPF removed all of these newlines?

Reply:
Could you confirm whether YAPF changed these newlines?  YAPF seems to be passing, but it doesn't match the rest of the codebase so I'm a little confused.  There are other open PRs with tests passing that don't have these changes, so maybe YAPF doesn't actually care about them?  The style guide (and rest of the code base) generally shows having blank lines between the different sections, so I'd prefer to keep them if YAPF isn't forcing us to change them: 
http://google.github.io/styleguide/pyguide.html#doc-function-raises